pkg/inspector/inspector_benchmark_test.go

```diff
@@ -30,3 +38,14 @@ func BenchmarkInspector_SingleSession_Send(b *testing.B) {
 		ins.Send(context.Background(), record.Record{Position: record.Position("test-pos")})
 	}
 }
+
+func BenchmarkInspector_10Sessions_Send(b *testing.B) {
+	ins := New(log.Nop(), 10)
+	for i := 0; i < 10; i++ {
+		ins.NewSession(context.Background(), "test-id")
+	}
+
```

Comment:
Calling `b.ResetTimer()` zeroes the elapsed benchmark time and memory allocation counters since the beginning of the test. This way, an expensive setup can be discarded from the test results. That will make the benchmark results more precise, you should add that here and in all other benchmarks that need a setup before the benchmark.

* https://pkg.go.dev/testing#B.ResetTimer